packages/device_info_plus/lib/device_info_plus_tizen.dart

```diff
@@ -10,8 +10,8 @@ import 'package:flutter/services.dart';
 ///
 /// See: https://docs.tizen.org/application/native/guides/device/system
 class TizenDeviceInfo {
-  /// Tizen device info class.
-  TizenDeviceInfo({
+  TizenDeviceInfo._({
```

Comment:
Technically this is a breaking change but I don't think there's any use case.

Reply:
Just curious! is there a reason why a named constructor is necessary here technically? kind of pattern?

Reply:
This just changes the visibility of the constructor. Any member that starts with `_` is considered private in Dart.